prow/plugins/verify-owners/verify-owners.go

```diff
@@ -38,8 +39,10 @@ import (
 
 const (
 	// PluginName defines this plugin's registered name.
-	PluginName     = "verify-owners"
-	ownersFileName = "OWNERS"
+	PluginName                    = "verify-owners"
```

Comment:
I really think we ought to make this a presubmit, rather than a plugin

Reply:
This is already a plugin in use.. it's also useful to have this as a plugin for other prow consumers.

Is there an advantage to it being a presubmit?

Reply:
How do problems identified here prevent merges?

IMO a failing context + job logs on spyglass are easier to understand and debug than this. 

We need to figure out how to get people to reuse jobs, which is the only reason for it to be a plugin -- and IMO not worth the extra label and logging complexity

Reply:
> We need to figure out how to get people to reuse jobs

Fwiw https://github.com/kubernetes/repo-infra was supposed to do this, but looks like that's not possible today.

Reply:
@fejta I'd love to make for portable jobs, without having to copy and maintain hack scripts in every repo (thinks like spelling, shellcheck, typecheck, etc). I'm not sure we have a good pattern for this, however.

Reply:
Feel free to drop the hold